### PR TITLE
Fix RememberCountered when replaced

### PR DESCRIPTION
**Pull request by tool4ever**
This is so you don't get to draw cards from _Swift Silence_ if you also control _Guile_.

### CODE REVIEW

**Review by kevlahnota — APPROVED** [review]

File: forge-gui/res/cardsfolder/h/heroism.txt [hunk 1]
```diff
@@ -1,11 +1,9 @@
 Name:Heroism
 ManaCost:2 W
 Types:Enchantment
-A:AB$ RepeatEach | Cost$ Sac<1/Creature.White/White Creature> | RepeatCards$ Creature.attacking+Red | UseImprinted$ True | RepeatSubAbility$ DBPump | SubAbility$ DBEffect | SpellDescription$ For each attacking red creature, prevent all combat damage that would be dealt by that creature this turn unless its controller pays {2}{R}.
-SVar:DBPump:DB$ Pump | RememberObjects$ Imprinted | UnlessCost$ 2 R | UnlessPayer$ ImprintedController | ShowCurrentCard$ Imprinted
-SVar:DBEffect:DB$ Effect | ReplacementEffects$ RPrevent | RememberObjects$ Remembered | ForgetOnMoved$ Battlefield | SubAbility$ DBCleanup
+A:AB$ RepeatEach | Cost$ Sac<1/Creature.White/White Creature> | RepeatCards$ Creature.attacking+Red | RepeatSubAbility$ DBEffect | SubAbility$ DBEffect | SpellDescription$ For each attacking red creature, prevent all combat damage that would be dealt by that creature this turn unless its controller pays {2}{R}.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/h/heroism.txt:4]: Hmm, I kind of like this better with just one Effect... was the old way causing a rules issue?

File: forge-gui/res/cardsfolder/upcoming/titania_natures_force.txt [hunk 2]
```diff
@@ -5,8 +5,8 @@ PT:6/6
 S:Mode$ Continuous | Affected$ Forest.YouOwn | MayPlay$ True | AffectedZone$ Graveyard | Description$ You may play Forests from your graveyard.
 T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Forest.YouCtrl | TriggerZones$ Battlefield | Execute$ TrigToken | TriggerDescription$ Whenever a Forest enters the battlefield under your control, create a 5/3 green Elemental creature token.
 SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ g_5_3_elemental | TokenOwner$ You
-T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Elemental.YouCtrl | OptionalDecider$ You | Execute$ TrigMill | TriggerDescription$ Whenever an Elemental you control dies, you may mill three cards.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/titania_natures_force.txt:8]: Seems like this should be OptionalDecider?

**Review by Northmoc — COMMENTED** [review]
Nice work

File: forge-gui/res/cardsfolder/h/heroism.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/h/heroism.txt:4]: Good point, will revert

File: forge-gui/res/cardsfolder/upcoming/titania_natures_force.txt (hunk 2 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/titania_natures_force.txt:8]: @Northmoc 
it must be on the Effect line

**Pull request merged**